pkg/action/rollback.go

```diff
@@ -211,7 +213,7 @@ func (r *Rollback) performRollback(currentRelease, targetRelease *release.Releas
 	}
 
 	deployed, err := r.cfg.Releases.DeployedAll(currentRelease.Name)
-	if err != nil {
+	if err != nil && err != driver.ErrReleaseNotFound {
```

Comment:
are you sure you want to ignore the not found errors here?

Reply:
Yea, the tests need to be fixed for this.

Reply:
Fixed per #7372's handling of mutable data.